### PR TITLE
[INLONG-9163][Agent] Delete trigger related file

### LINKED ISSUE #9163: [Improve][Agent] Delete trigger related file
### Description

Delete trigger related file

### InLong Component

InLong Agent

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by justinwwhuang**
### Prepare a Pull Request
[INLONG-9163][Agent] Delete trigger related file

- Fixes #9163 

### Motivation

Delete trigger related file

### Modifications

Delete trigger related file

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/HeartbeatManager.java [hunk 1]
```diff
@@ -181,34 +172,24 @@ private HeartbeatMsg buildHeartbeatMsg() {
...
-            streamHeartbeat.setStatus(status);
-            streamHeartbeats.add(streamHeartbeat);
-        });
-
-        heartbeatMsg.setGroupHeartbeats(groupHeartbeats);
-        heartbeatMsg.setStreamHeartbeats(streamHeartbeats);
-
-        LOGGER.info("heartbeat jobIds {} heartbeatMsg {}", jobIds, heartbeatMsg);
+        /*
+         * Map<String, JobWrapper> jobWrapperMap = taskManager.getJobs(); List<GroupHeartbeat> groupHeartbeats =
```
Comment (EMsnap) [inline inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/HeartbeatManager.java:176]: please remove this

File: inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/task/PositionManager.java [hunk 2]
```diff
@@ -44,14 +44,14 @@ public class PositionManager extends AbstractDaemon {
...
-    private final JobProfileDb jobConfDb;
+    private JobProfileDb jobConfDb;
     private final AgentConfiguration conf;
     private ConcurrentHashMap<String, ConcurrentHashMap<String, Long>> jobTaskPositionMap;
 
     private PositionManager(AgentManager agentManager) {
         this.conf = AgentConfiguration.getAgentConf();
         this.agentManager = agentManager;
-        this.jobConfDb = agentManager.getJobManager().getJobConfDb();
+        // this.jobConfDb = agentManager.getJobManager().getJobConfDb();
```
Comment (EMsnap) [inline inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/task/PositionManager.java:54]: please remove this

File: inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/HeartbeatManager.java (hunk 1 above)
Comment (justinwwhuang) [reply inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/HeartbeatManager.java:176]: get, fix it next issue

File: inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/task/PositionManager.java (hunk 2 above)
Comment (justinwwhuang) [reply inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/task/PositionManager.java:54]: get, fix it next issue

**Review by doleyzi — APPROVED** [review]

**Review by EMsnap — DISMISSED** [review]

**Review by luchunliang — DISMISSED** [review]

**Review by EMsnap — APPROVED** [review]

**Review by luchunliang — APPROVED** [review]

**Pull request merged**